Fixes build failure on glibc-2.23

../video/out/opengl/lcms.c: In function 'get_vid_profile':
../video/out/opengl/lcms.c:235:27: error: implicit declaration of function 'pow' [-Werror=implicit-function-declaration]
             double binv = pow(src_black[i], 1.0/gamma);
                           ^
diff --git a/video/out/opengl/lcms.c b/video/out/opengl/lcms.c
index 7db8da6..3eb3d53 100644
--- a/video/out/opengl/lcms.c
+++ b/video/out/opengl/lcms.c
@@ -17,2 +17,3 @@
 
+#include <math.h> /* pow() */
 #include <string.h>
